src/rtextures.c

```diff
@@ -3979,22 +4017,29 @@ Texture2D LoadTexture(const char *fileName)
 
 // Load a texture from image data
 // NOTE: image is not unloaded, it must be done manually
-Texture2D LoadTextureFromImage(Image image)
-{
-    Texture2D texture = { 0 };
+Texture2D LoadTextureFromImage(Image image) {
+  Texture2D texture = {0};
```

Comment:
Hello,
Please revert the changes on `LoadTextureFromImage()` as it breaks current [code conventions](https://github.com/raysan5/raylib/wiki/raylib-coding-conventions).
Best regards.